TypeScript.YAML-tmLanguage

```diff
@@ -169,7 +160,7 @@ repository:
       '5': { name: keyword.operator.ts }
       '6': { name: entity.name.function.ts }
       '7': { name: keyword.operator.ts }
-    end: '(?=\}|;|,|$)|(?<=\})'
```

Comment:
Hmm, the thing is, a method declaration might not have a body and might just need on a newline. For instance,

```ts
abstract class C {
    abstract foo()
    abstract bar()
}
```

or with a method signature:

```ts
interface I {
    foo(): number
    bar(): string;
}